openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -401,16 +408,13 @@ def build_layout(project_name, asset_name):
         containers = bpy.context.scene.openpype_containers
 
         # Link loaded containers to layout collection
-        for c in containers:
-            if not isinstance(c.outliner_entity, bpy.types.Collection):
-                continue
-            if c.outliner_entity not in layout_container.children.values():
-                layout_container.children.link(c.outliner_entity)
-            if (
-                c.outliner_entity
-                in bpy.context.scene.collection.children.values()
-            ):
-                bpy.context.scene.collection.children.unlink(c.outliner_entity)
+        for container in containers:
+            for root in container.get_root_outliner_datablocks():
+                if not isinstance(root, bpy.types.Collection):
```

Comment:
Here it looks like you've inverted the condition. Before it was continuing if not Collection, now it executes only if not Collection. I bet it is a mistake and you want to process only for Collection (cf my previous comment, you'll not need this any more)

Reply:
Ouch indeed !